templates/webapi/test/tr_job_result.html.ep

```diff
@@ -37,5 +37,6 @@
             %= include 'test/tr_job_result_failedmodules', jobid => $jobid, failedmodules => $res->{failures}, resultid => $resultid
          % }
          %= include 'test/tr_job_result_details', jobid => $jobid, res => $res
+         <span class='dependency' data="<%= $res->{deps} %>" jobid="<%= $jobid %>" result="<%= $res->{overall} %>"></span>
```

Comment:
doesn't this include the span element for *all* job results? We should only add it for jobs that are part of a cluster and also I would avoid that by default for child jobs without any parallel jobs, e.g. a "extratests_in_textmode" that is simply loading a qcow2 image created by a parent.

Reply:
yes, all job results td will include this span. The check code that whether a job has dependencies is in javascript, to reuse the same code,  and not add to much check code in template, so I guess it doesn't matter.

Reply:
ok, so but will any icon show up for the following cases:

1. job not part of any cluster, i.e. no parallel jobs and no kids and no parents
2. job is child jobs without any parallel jobs, e.g. a "extratests_in_textmode"

Reply:
1. If the job not part of any cluster, no icon show up.
2. if the job is a child job (I think it has a setting `START_AFTER_TEST`, and not `PARALLEL_WITH`), the icon will show up. I think it is expected.
For example:
![2](https://user-images.githubusercontent.com/11713397/88913693-0c34c880-d294-11ea-9361-c5151c367c41.png)
job `fips_ker_mode_tests_crypt_core` has a setting `START_AFTER_WITH=xxx`
job `fips_ker_mode_tests_crypt_install` has no any dependency.

Reply:
I mentioned the example "extra tests" which does not have an icon on /tests . I prefer to have the same here. Otherwise there are IMHO just too many icons with too little information content

Reply:
Here is a screenshot I take from OSD page
![3](https://user-images.githubusercontent.com/11713397/88915645-42c01280-d297-11ea-816f-7fc1ac2ecdf9.png)
These jobs in the picture all have a setting `START_AFTER_TEST= `, is it the same to your example 'extra tests' ? they all have the icon.

Reply:
yeah, you are right. I should have taken a screenshot of what I observed because I do not know anymore :D Ok, I still think that this might be a bit too much information but we should leave your PR as is for now